cmd/sanitize-prow-jobs/main.go

```diff
@@ -15,27 +17,66 @@ import (
 	"sigs.k8s.io/yaml"
 )
 
-const defaultCluster = "api.ci"
-
 type options struct {
 	prowJobConfigDir string
+	configPath       string
 
 	help bool
 }
 
+type Config struct {
+	Clusters struct {
+		Default      string `json:"default"`
+		Distribution map[string]struct {
+			Jobs   []string `json:"jobs"`
+			Files  []string `json:"files"`
+			Groups []string `json:"groups"`
+		} `json:"distribution"`
+		Groups map[string][]string       `json:"groups"`
+		REMap  map[string]*regexp.Regexp `json:"-"`
+	} `json:"clusters"`
+}
+
 func bindOptions(flag *flag.FlagSet) *options {
 	opt := &options{}
 
 	flag.StringVar(&opt.prowJobConfigDir, "prow-jobs-dir", "", "Path to a root of directory structure with Prow job config files (ci-operator/jobs in openshift/release)")
+	flag.StringVar(&opt.configPath, "config-path", "", "Path to the config file (core-services/sanitize-prow-jobs/_config.yaml in openshift/release)")
 	flag.BoolVar(&opt.help, "h", false, "Show help for ci-operator-prowgen")
 
 	return opt
 }
 
-func determinizeJobs(prowJobConfigDir string) error {
+func determinizeJobs(prowJobConfigDir, configPath string) error {
 	errChan := make(chan error)
-
 	var errs []error
+
+	config := &Config{}
+	data, err := ioutil.ReadFile(configPath)
+	if err != nil {
+		errs = append(errs, err)
```

Comment:
I think we should return the error here and in the unmarshaling, as all forthcoming errors may be a result of erroring here. Also please wrap them so they are easier to understand and can be grepped for in the codebase

Reply:
Done.